zun/container/k8s/driver.py

```diff
@@ -882,3 +908,23 @@ def create_capsule(self, context, capsule, **kwargs):
 
     def delete_capsule(self, context, capsule, **kwargs):
         raise NotImplementedError()
+
+    def get_secrets_for_image(self, image, context):
```

Comment:
This is technically a private method, can you name it `_get_secrets_for_image`? This driver has a huge public API already b/c it inherits from the base Zun implementation and I'd like to keep anything K8s-driver-specific to clearly-marked private methods for clarity.